components/proxy/conf/Caddyfile

```diff
@@ -210,7 +210,7 @@ https://{$GITPOD_DOMAIN} {
 	@codesync path /code-sync*
 	handle @codesync {
 		gitpod.cors_origin {
-			base_domain {$GITPOD_DOMAIN}
+			any_domain true
```

Comment:
@filiptronicek Could you elaborate on the motivation to allow accessing this endpoint from all domains? :thinking:

Reply:
because VS Code Desktop does not provide origin, but only bearer auth

Reply:
Thx for the explanation, make sense!